data/ciweimao

```diff
@@ -0,0 +1,2 @@
+ciweimao.com
+kuangxiangit.com
```

Comment:
can not tell whether this site is owned by the `ciweimao` from the whois info
resources from this site were loaded when accessing the `ciweimao` is not solid proof.

Reply:
![image](https://user-images.githubusercontent.com/13471233/220835850-b628f827-1232-4110-a5af-479be76bfb1a.png)